jpsreport/scripts/plot_Nt.py

```diff
@@ -0,0 +1,81 @@
+# run script without arguments for help
+import sys
+import os
+from xml.dom.minidom import parse
+import glob
+import argparse
+import numpy as np
+import matplotlib.pyplot as plt
+
+def getParserArgs():
+        parser = argparse.ArgumentParser(description='Plot N-t graph measured by method A')
+        parser.add_argument("-f", "--inifile", help='jpsreport inifile (required)', required=True)
+        parser.add_argument("-i", "--id", help='id of the measurement area (optional)', nargs='?', default=-1, type=int)
+        args = parser.parse_args()
+        return args
+
+def plot_nt(measurement_id):
+    """
+    plot N(t) curves (Method A)
+    args:
+    measurement area id (int)
+    """
+    legend = False
```

Comment:
`legend = False`: Can I parse it (line 10-15)?

Reply:
Not necessary.

Reply:
Why?

Reply:
Because it's not necessary.

Reply:
The file was removed with commit a8496940. The question is therefore invalid. I will resolve this conservation.